Topics/Development_Process/App_Store_Deployment.md

```diff
@@ -0,0 +1,144 @@
+
+The benefit of using Ad hoc is that you would not need the app to be reviewed before distributing it out for testing. This means that you would be able to get your app to your testers immediately and possibly avoid getting the app rejected during the review process.
+
+#### Distribution Process
+After your generate all your needed certificates, you would then need to generate your Provisioning Profile, selecting the app you wish to use as well as all the devices you want to access it. 
+
+You would then proceed to complete the rest of the fields on the provisioning profile page and download the profile to import onto Xcode to build the app.
+
+Now you can decide how you wouild like distribute the app, where you can choose between the following options for distribution:
+- App Store Connect (TestFlight/App Store)
+- Ad Hoc
+- Enterprise
+- Development
+
+Should you choose TestFlight in App Store Connect, your process would be simple and you can follow further directions online on how your testers can download your app on the TestFligth app market. 
+Following the steps would allow you to easily upload your .ipa onto App Store Connect where you can manage your next steps online.
+
+If you chose to use Ad Hoc, however, you would build a .ipa which you can send to your testers to install onto their device or device simulator on Xcode.
+
+For further clarification about App Distribution, you can refer to these **Apple's Documentation** links below:
+
+- [Distributing your app for beta testing and releases](https://developer.apple.com/documentation/xcode/distributing-your-app-for-beta-testing-and-releases)
+
+- [Distributing your app to registered devices](https://developer.apple.com/documentation/xcode/distributing-your-app-to-registered-devices)
+
+### App Store Submission 
+There will be a full review process, which should be completed within 24 hours according to Apple, but it may take longer depending on your app.
+In comparison to TestFlight's review process, since this would go to the official App Store, they will be enforcing stricter standards.
+
+It would be good practice to continously develop your app keeping proper safety and privacy guidelines in mind, but in case you would like to assess on your own prior to submission, you could refer to these specific review guidlines:
+
+- [App Store Review Guidelines](https://developer.apple.com/app-store/review/guidelines/)
+
+- [Human Interface Design Guidelines](https://developer.apple.com/design/human-interface-guidelines/)
+
+- [Guidelines for Using Apple Trademarks and Copyrights](https://www.apple.com/legal/intellectual-property/guidelinesfor3rdparties.html)
+
+
+If you used TestFlight for beta testing earlier, you should be familiar with this process as it is very similar for the App Store Review. If you used Ad hoc to test, then you may refer to the steps mentioned in the TestFlight process in getting the .ipa uploaded onto App Store Connect.
```

Comment:
Would be convenient if you linked the section instead of just mentioning it so that they don't get lost trying to navigate back and forth.

Reply:
Good idea, I didn't even think about linking to one of my own sections. That'll definitely make things easier.